components/AuthorBox.tsx

```diff
@@ -62,10 +88,51 @@ const AuthorBox: React.ElementType<AuthorBoxProps> = ({
             borderRadius: 100 / 2,
           }}
         />
+        <View
+          style={{
+            flexDirection: "row",
+            display: "flex",
+            justifyContent: "center",
+            marginLeft: 12.5,
+          }}
+        >
+          {(additionalAuthorInfo?.email && !additionalAuthorInfo.email.endsWith("@stanford.edu")) || 
```

Comment:
We can remove the `.endsWith` logic here per the above change

Reply:
Done

Reply:
Wait where is the above change? Is it on the backend?

Reply:
The change mentioned above changing `email` field to `dailyEmail`, yeah

Reply:
Also I think we need to remove the entire second half of the && right

Reply:
Yeah, that's what I meant by the `.endsWith` logic, sorry